roles/hello-world-apb/defaults/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+app: hello-world
+apb_id: 0
+app_name: "{{ app }}-{{ apb_id }}"
```

Comment:
My hope is to have the broker provide the service instance id as `apb_id` instead of `_apb_service_instance_id` (and have `apb_id` passed on all APB actions) to make this work with and w/o a broker. If `_apb_service_instance_id` stays, then I'll change this.